nss-day-cohort-39/tabloid-mvc-the-cure
# Add + Remove Tags to and fro Posts

**Changes/Testing Steps
- Added edit and delete functionality to MyPosts (details does not yet show tags)
- Added PostTag Controller, Repo, Model, TagIndexViewModel, and View
- Added additional methods to TagRepo specifically for associated tags views


**Testing
- Go to MyPosts and Edit/Delete a Post
- When choosing Posts, a list of associated tags with each posts is now shown in Post Details
- Choose manage tags and select which tags you want and which tags you don't
- Then click save and be directed back to Post Details Page
- Changes to tags you chose should appear in the list of tags
